public/assets/css/egeeio.css

```diff
@@ -13,6 +13,9 @@ div.gmod_256 {
 div.quake_256 {
   background: url('../images/servers/quake_256.jpg') center / cover;
 }
+div.quake2_256 {
```

Comment:
You even dug into the CSS!? :dancers: In the future, this may change because we'll be using Less instead of raw css. It'll make this process easier.